docs/writing-tests/accessibility-testing.md

```diff
@@ -148,35 +156,61 @@ Disable accessibility testing for stories or components by adding the following
 
 <!-- prettier-ignore-end -->
 
-## Automate accessibility tests with Jest
+## Automate accessibility tests with test runner
 
-Accessibility testing with Storybook shortens the feedback loop which means you fix issues faster. Reuse stories in a Jest test and run an accessibility audit on them using the [jest-axe integration](https://github.com/nickcolley/jest-axe). That also unlocks the ability to integrate accessibility tests into your functional testing pipeline.
+Accessibility testing with Storybook shortens the feedback loop, which means you fix issues faster. You can then integrate these accessibility tests into your test automation pipeline using the Storybook [test runner](./test-runner.md#test-hook-api-experimental) and [axe-playwright](https://github.com/abhinaba-ghosh/axe-playwright).
 
-For example, include the following test file to run an accessibility test on a story:
+### Setup
+
+To enable accessibility testing with the test runner, you will need to take additional steps to set it up properly. Detailed below is our recommendation to configure and execute them.
+
+Run the following command to install the required dependencies.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'react/accessibility-testing-with-jest-axe.js.mdx',
-    'vue/accessibility-testing-with-jest-axe.js.mdx',
+    'common/storybook-test-runner-axe-playwright.yarn.js.mdx',
+    'common/storybook-test-runner-axe-playwright.npm.js.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-When you execute your test script, it will run the accessibility audit along with any interaction tests you might have.
+Add a new [configuration file](./test-runner.md#test-hook-api-experimental) inside your Storybook directory with the following inside:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-test-runner-a11y-config.js.mdx',
+    'common/storybook-test-runner-a11y-config.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<div class="aside">
+
+💡 `preRender` and `postRender` are convenient hooks that allow you to extend the test runner's default configuration. They are **experimental** and subject to changes. Read more about them [here](./test-runner.md#test-hook-api-experimental).
+
+</div>
+
+By default, Axe assumes that you're testing a page and checks whether you've specified an `<h1>` and `<main>`. However, most of your stories are for components and not pages. That's why we use the `context.title.includes('Page')` check to [enable/disable](https://github.com/abhinaba-ghosh/axe-playwright#context-optional) Axe's page-level rules.
+
+When you execute the test runner (i.e., `yarn test-storybook`), it will run the accessibility audit and any [interaction tests](./interaction-testing.md) you might have configured for each component story.
```

Comment:
> i.e., `yarn test-storybook`

My understanding is that "i.e.", "e.g.", and other Latin abbreviations are not well understood by international audiences.

Reply:
This was addressed in the latest commits. Marking it as solved.